skimage/exposure/hdr.py

```diff
@@ -0,0 +1,351 @@
+import numpy as np
+import warnings
+
+
+def make_hdr(images, exposure, radiance_map, depth=16, channel_axis=None):
+    """
+    Compute the HDR image from a series of images with a given radiance
+    mapping.
+
+    Parameters
+    ----------
+    images: list of numpy arrays, or numpy ndarray or ImageCollection
+        List of images in the form of numpy arrays, or a numpy array with the
+        first dimension being the different images.
+        Either greyscale or colour (RGB). Can't be float images
+        List of images as numpy arrays. Either greyscale or colour
+        (RGB).
+        Can also be an ImageCollection
+    exposure : numpy 1D array
+        Array of exposure times in seconds.
+        Images some times have these in the exif information.
+    radiance_map : numpy array
+        Array (idx) mapping counts to radiance value, if input is RGB this must
+        be Nx3. See `get_crf`.
+    depth : int, optional
+        Pixel depth.
+    channel_axis : int or None, optional
+        If None, the images are assumed to be greyscale images.
+        Otherwise, this parameter indicates which axis of the array corresponds
+        to channels. This includes the list index, meaning the images
+        with "normal" axis (X, Y, C) should set `channel_axis=3`
+
+    Returns
+    -------
+    hdr : numpy array
+        The HDR image, either greyscale or RGB depending on input, in ln(E).
```

Comment:
```suggestion
        The HDR image, either greyscale or RGB depending on input, in ln(E).
```

We should clarify here what `ln(E)` means.